Bug: Override called on non-virtual member function

### Current Behaviour

getting this when i'm trying to build azerothcore with your module installed

/home/azeroth/azerothcore/modules/mod-solo-lfg/src/Lfg_Solo.cpp:21:34: fatal error: only virtual member functions can be marked 'override'
    void OnLogin(Player* player) override
                                 ^~~~~~~~
1 error generated.
[ 46%] Building CXX object src/server/game/CMakeFiles/game.dir/Entities/Creature/GossipDef.cpp.o
make[2]: *** [modules/CMakeFiles/modules.dir/build.make:580: modules/CMakeFiles/modules.dir/mod-solo-lfg/src/Lfg_Solo.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1562: modules/CMakeFiles/modules.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

### Expected Behaviour

not this.

### Steps to reproduce the problem

build acore

### Extra Notes

_No response_

### AC rev. hash/commit

fresh clone

### Operating system

ubuntu server 22.04

### Custom changes or Modules

mod-solo-lfg
mod-ah-bot
mod-eluna

--- a/src/Lfg_Solo.cpp
+++ b/src/Lfg_Solo.cpp
@@ -18,7 +18,7 @@ class lfg_solo_announce : public PlayerScript
 public:
     lfg_solo_announce() : PlayerScript("lfg_solo_announce") {}
 
-    void OnLogin(Player* player) override
+    void OnPlayerLogin(Player* player) override
     {
         // Announce Module
         if (sConfigMgr->GetOption<bool>("SoloLFG.Announce", true))
@@ -27,7 +27,7 @@ public:
         }
     }
 
-    void OnRewardKillRewarder(Player* /*player*/, KillRewarder* /*rewarder*/, bool isDungeon, float& rate) override
+    void OnPlayerRewardKillRewarder(Player* /*player*/, KillRewarder* /*rewarder*/, bool isDungeon, float& rate) override
     {
         if (!isDungeon
             || !sConfigMgr->GetOption<bool>("SoloLFG.Enable", true)

